### ISSUE TITLE
Detaching fails when you have a pivot relation without soft deleting in the same model as a relation that does

### ISSUE THREAD

**Issue by alistairreynolds** [ddzobov/laravel-pivot-softdeletes#1]
Even though I'm not using the `->softDeletes()` method with a `backlistedSites` relation, it still tries to use it when I call `detach` on the relation. Syncing/adding is fine, just not detaching.

In the example below, the `cat` request attribute is from an array of checkboxes

## Error

From **/ddzobov/laravel-pivot-softdeletes/src/Relations/Concerns/InteractsWithPivotTable.php**

```SQLSTATE[42S22]: Column not found: 1054 Unknown column '' in 'field list' (SQL: update `map_type_tag_blacklist` set `` = 2020-02-07 08:50:58 where `tag_id` = 1)```

  In line 76: `$results = $query->update($attributes);`

## Saving method

```
public function process(\App\Tag $tag, \App\Http\Requests\TagsRequest $request)
{
    $tag->fill($request->except('cat'));
    if($request->has('cat')) {
        $tag->blacklistedCategories()->syncWithoutDetaching($request->get('cat'));
    }else{
        $tag->blacklistedCategories()->detach();
    }
    return $tag->save();
}
```

## Model

```
class Tag
{
    public function sites()
    {
        return $this
            ->belongsToMany(Site::class)
            ->using(SiteTag::class)
            ->whereNull('client_id')
            ->withSoftDeletes();
    }


    public function blacklistedCategories()
    {
        return $this->belongsToMany(SiteCategory::class, 'map_type_tag_blacklist', 'tag_id', 'map_type_id');
    }
}
```

Loving the package though! For now I can get around it by adding soft deletes to my blacklistedCategories relation, which seems to work.

**Comment by ddzobov** [comment]
Hello!

Just published 1.1.0, check please, maybe this error was fixed in this release.

**Comment by alistairreynolds** [comment]
That all works perfect now - thanks!
